timeutil/schedule.go

```diff
@@ -119,8 +119,10 @@ func (w Week) String() string {
 	return day + strconv.Itoa(int(w.Pos))
 }
 
-// WeekSpan represents a span of weekdays between Start and End days. WeekSpan
-// may wrap around the week, eg. fri-mon is a span from Friday to Monday
+// WeekSpan represents a span of weekdays between Start and End days, which may
+// be a single day. WeekSpan may wrap around the week, eg. fri-mon is a span
+// from Friday to Monday, mon1-fri is a span from the first Monday to the
+// following Friday, while mon1-mon1 represents the 1st Monday of a month.
```

Comment:
The comment refers to the internal representation, I've clarified it though.

As for rejecting `mon1-mon1`, since it's really a `mon1` schedule, I'll push it in a follow up.